src/main/java/org/dbflute/intro/app/web/playsql/data/PlaysqlDataAction.java

```diff
@@ -0,0 +1,39 @@
+package org.dbflute.intro.app.web.playsql.data;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+import javax.annotation.Resource;
+
+import org.dbflute.intro.app.logic.intro.IntroPhysicalLogic;
+import org.dbflute.intro.app.web.base.IntroBaseAction;
+import org.lastaflute.web.Execute;
+import org.lastaflute.web.response.JsonResponse;
+
+/**
+ * @author prprmurakami
+ */
+public class PlaysqlDataAction extends IntroBaseAction{
+
+	// ===================================================================================
+    //                                                                           Attribute
+    //                                                                           =========
+    @Resource
+    private IntroPhysicalLogic introPhysicalLogic;
+
+    @Execute(urlPattern = "{}/@word")
+    public JsonResponse<Void> data(String clientName) {
+    	File dataDir = new File(introPhysicalLogic.buildClientPath(clientName, "playsql", "data"));
```

Comment:
Create `PlaysqlDataLogic`, move this logic.

Reply:
Like a `PlaysqlMigrationAction` .

Reply:
@EgumaYuto 
>Create PlaysqlDataLogic, move this logic.
>Like a PlaysqlMigrationAction .

Fixed.
c7fd4ce